gridcapa-cse-api/src/main/java/com/farao_community/farao/gridcapa_cse/api/resource/CseRequest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa_cse.api.resource;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Type;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Type("cse-request")
+public class CseRequest {
+    @Id
+    private String id;
+    private final String instant;
+    private String cgmUrl;
+    private String mergedCracUrl;
+    private String mergedGlskUrl;
+    private String ntcReductionsUrl;
+    private String ntc2AtItUrl;
+    private String ntc2ChItUrl;
+    private String ntc2FrItUrl;
+    private String ntc2SiItUrl;
+    private String targetChUrl;
+    private String vulcanusUrl;
+    private String yearlyNtcUrl;
+
+    @JsonCreator
+    public CseRequest(@JsonProperty("id") String id,
+                      @JsonProperty("instant") String instant,
+                      @JsonProperty("cgmUrl") String cgmUrl,
+                      @JsonProperty("mergedCracUrl") String mergedCracUrl,
+                      @JsonProperty("mergedGlskUrl") String mergedGlskUrl,
+                      @JsonProperty("ntcReductionsUrl") String ntcReductionsUrl,
+                      @JsonProperty("ntc2AtItUrl") String ntc2AtItUrl,
+                      @JsonProperty("ntc2ChItUrl") String ntc2ChItUrl,
+                      @JsonProperty("ntc2FrItUrl") String ntc2FrItUrl,
+                      @JsonProperty("ntc2SiItUrl") String ntc2SiItUrl,
+                      @JsonProperty("targetChUrl") String targetChUrl,
+                      @JsonProperty("vulcanusUrl") String vulcanusUrl,
+                      @JsonProperty("yearlyNtcUrl") String yearlyNtcUrl) {
+        this.id = id;
+        this.instant = instant;
+        this.cgmUrl = cgmUrl;
+        this.mergedCracUrl = mergedCracUrl;
+        this.mergedGlskUrl = mergedGlskUrl;
+        this.ntc2AtItUrl = ntc2AtItUrl;
+        this.ntc2ChItUrl = ntc2ChItUrl;
+        this.ntc2FrItUrl = ntc2FrItUrl;
+        this.ntc2SiItUrl = ntc2SiItUrl;
+        this.ntcReductionsUrl = ntcReductionsUrl;
+        this.targetChUrl = targetChUrl;
+        this.vulcanusUrl = vulcanusUrl;
+        this.yearlyNtcUrl = yearlyNtcUrl;
+    }
+
+    public CseRequest(@JsonProperty("id") String id,
```

Comment:
For IDCC there is all the NTC2 files but no targetCH and for D2CC there are no NTC2 files but a target CH file. Here it does not match.
You can actually create two static creators CseRequest.idccProcess(***IDCC files***) and CSERequest.d2ccProcess(***D2CC files***)